### PR TITLE
fix issue #2 regarding error 500 when searching products

### PR DESCRIPTION
**Pull request by Joukz**

### CODE REVIEW

File: controllers/admin/AdminAjaxPs_buybuttonliteController.php [hunk 1]
```diff
@@ -92,7 +92,15 @@ public function getSqlQuery($query, $id_lang)
...
     {
         $product = new Product($id_product);
+
         $link_rewrite = $product->link_rewrite;
+        if (is_array($link_rewrite)) {
+            $i = 0;
+            while (empty($product->link_rewrite[$i])) {
+                $i++;
+            }
+            $link_rewrite = $product->link_rewrite[$i];
```
Comment (PierreRambaud) [inline controllers/admin/AdminAjaxPs_buybuttonliteController.php]: I don't understand what you are trying to do here? getting the last element in link_rewrite?

File: controllers/admin/AdminAjaxPs_buybuttonliteController.php [hunk 2]
```diff
@@ -119,7 +127,15 @@ public function getProductAttributeImage($id_product, $id_product_attribute)
...
+        }
```
Comment (PierreRambaud) [inline controllers/admin/AdminAjaxPs_buybuttonliteController.php]: same as previous comment.

**Comment by marionf** [comment]
I still have issue #4 

Here the link is ok: 

capture d ecran_384

If I click on Copy to clipboard and past it, it's not: 

capture d ecran_383

**Comment by marionf** [comment]
@Joukz 
I am logged in FO, I select a product, I select go to cart and click on Copy to clipboard, I am redirected to the home page and my cart is empty

**Comment by marionf** [comment]
@Joukz 

I have a customizable product with required field and don't see any alert in the module

capture d ecran_474

**Comment by marionf** [comment]
@Joukz 

I see the message if the product has a customizable field not required.
The message should be displayed only if there is at least one required field.

File: app/src/App.vue [hunk 3]
```diff
@@ -99,6 +106,7 @@ import PsCard from '@/components/PsCard.vue'
 import PsCardHeader from '@/components/PsCardHeader.vue'
 import PsCardBlock from '@/components/PsCardBlock.vue'
 import PsCardFooter from '@/components/PsCardFooter.vue'
+import Alert from '@/components/Alert.vue'
```
Comment (PierreRambaud) [inline app/src/App.vue]: You're also able to write `@components` :)

**Comment by PierreRambaud** [comment]
Thanks @Joukz

**Pull request merged**